rust/chains/abacus-ethereum/src/inbox.rs

```diff
@@ -5,10 +5,11 @@ use std::{error::Error as StdError, sync::Arc};
 
 use async_trait::async_trait;
 use ethers::contract::abigen;
-use ethers::core::types::{H256, U256};
+use ethers::prelude::*;
 use eyre::Result;
 use tracing::instrument;
 
+use crate::trait_builder::MakeableWithProvider;
```

Comment:
nit: should this go next to the `use crate` on line 19?

Reply:
Uh, probably. I try to run the "import optimizer" in my IDE to sort these but I don't always remember.

Reply:
Honestly I don't care _that much_ if imports are perfectly organized as long as there are not unused ones or stupid wildcards.